scripts/newinstall.sh

```diff
@@ -165,7 +165,7 @@ fi
 ##########	Test/warn about Python versions, offer to get miniconda2 if too old
 
 if true; then
-	PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
+	PYVEROK=$($PYTHON -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
```

Comment:
My knee jerk thinking is that it's better to be conservative at this point until we're sure the entire stack is python3 compatible.  I think the target audience for {{newinstall.sh}} is end-users rather than full time devs.